src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1873,6 +1889,7 @@ public virtual void Dispose()
             if (_disposeOnceRunner.Disposed)
                 return;
 
+            GC.SuppressFinalize(this);
             _disposeOnceRunner.Dispose();
```

Comment:
Should probably be in the `_disposeOnceRunner` action.
Note that we don't need to check, safe to call multiple time to `_disposeOnceRunner.Dispose()`

Reply:
Removed GC.SuppressFinalize() as multiple Dispose call situation handled by _disposeOnceRunner.